store/src/cli/info.rs

```diff
@@ -70,9 +70,26 @@ fn get_timestamp(file_type: &FileType, buf: &[u8]) -> Result<DateTime<Utc>> {
             SpeedtestReqV1::decode(buf).map(|entry| datetime_from_epoch(entry.timestamp))?
         }
         FileType::CellHeartbeatIngestReport => CellHeartbeatIngestReportV1::decode(buf)
-            .map(|entry| datetime_from_epoch(entry.report.unwrap().timestamp))?,
+            .map_err(|error| error.into())
+            .and_then(|ingest_report| {
+                ingest_report.report.ok_or_else(|| {
+                    Error::Custom(
+                        "CellHeartbeatIngestReportV1 does not contain a CellHeartbeatReqV1"
+                            .to_string(),
+                    )
+                })
+            })
+            .map(|heartbeat_req| datetime_from_epoch(heartbeat_req.timestamp))?,
         FileType::CellSpeedtestIngestReport => SpeedtestIngestReportV1::decode(buf)
-            .map(|entry| datetime_from_epoch(entry.report.unwrap().timestamp))?,
+            .map_err(|error| error.into())
```

Comment:
I think you can do `.map_err(Error::from)` for a better shorthand?

Reply:
Fixed af297e3 and thank you, reads better